### DISCUSSION TITLE
blog/south-korea-trip/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#364] [Announcements]
# blog/south-korea-trip/

韩国旅行体验：从首尔的繁华与文化冲突，到釜山的宁静与活力，再到仁川的历史沉淀。

https://jimmysong.io/blog/south-korea-trip/

**Comment by feiandxs** [comment]
想要那种景区和日常生活还能融汇在一起的地方，而不是把一个过去的东西划出来保护好成为一个景点，这样总觉得旅游的行程会变得隔靴搔痒。

就感觉自己只是去了一个拍照的点而不是一个真正的“地方”。

**Reply by rootsongjc** [reply]
我觉得可能只有融入到当地民众当中，听他们讲述的历史才是最直接和生动的。
